prime-router/src/testIntegration/kotlin/datatests/MappingInventoryTests.kt

```diff
@@ -0,0 +1,79 @@
+package gov.cdc.prime.router.datatests
+
+import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.cli.tests.CompareData
+import org.junit.jupiter.api.Test
+
+class MappingInventoryTests {
```

Comment:
So in the inverse of what we were talking about yesterday, I think this file is going to get huge if we put all the tests here?  Maybe just rename the file to XTN?